src/mongoParser/whereClause.js

```diff
@@ -39,17 +39,25 @@ function parseWhere(whereTree) {
     return +whereTree.value;
   case "Boolean":
     return whereTree.value.toLowerCase() === "true";
+  case "Null":
+    return null;
```

Comment:
Please add a test for the `null` scenario

Reply:
Thanks, added a test to my latest commit.